fighters/pichu/src/acmd/specials.rs

```diff
@@ -2,104 +2,333 @@
+        }
+        frame(lua_state, 5.0);
+        if is_excute(fighter) {
+            AttackModule::clear_all(boma);
+            FT_MOTION_RATE(fighter, 1.5);
+        }
     }
-    
 }
-
-#[acmd_script( agent = "pichu", script = "game_specialairlwhit" , category = ACMD_GAME , low_priority)]
-unsafe fn pichu_special_air_lw_hit_game(fighter: &mut L2CAgentBase) {
+#[acmd_script( agent = "pichu", scripts = ["effect_speciallwhit", "effect_specialairlwhit"] , category = ACMD_EFFECT , low_priority)]
+unsafe fn pichu_special_lw_hit_effect(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
+    frame(lua_state, 1.0);
+    if is_excute(fighter) {
+        EFFECT_FOLLOW_NO_STOP(fighter, Hash40::new("pichu_cheek"), Hash40::new("head"), 0, 0, 0, 0, -90, -90, 1, true);
+        LANDING_EFFECT(fighter, Hash40::new("sys_down_smoke"), Hash40::new("top"), 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, false);
+    }
+    frame(lua_state, 1.0);
+    WorkModule::is_flag(boma, *FIGHTER_PIKACHU_STATUS_WORK_ID_FLAG_KAMINARI_ATTACK_HIT);
+    if is_excute(fighter) {
+        EFFECT_FOLLOW_NO_STOP(fighter, Hash40::new("pichu_kaminari_hit2"), Hash40::new("top"), 0, -2, 0, 0, 90, 0, 1.15, true);
+        EFFECT_FOLLOW_NO_STOP(fighter, Hash40::new("pichu_kaminari_hit"), Hash40::new("top"), 0, -2, 0, 0, 90, 0, 0.85, true);
+        if VarModule::is_flag(fighter.battle_object, vars::pichu::status::IS_CHARGE_ATTACK) {
+            EFFECT_FOLLOW_NO_STOP(fighter, Hash40::new("pichu_final_explosion"), Hash40::new("top"), 0, 5, 0, 0, 0, 0, 0.5, false);
+        }
+    }
+    if is_excute(fighter) {
+        FLASH(fighter, 0, 0, 0, 0);
+        BURN_COLOR(fighter, 2, 2, 0.5, 0.9);
+    }
+    wait(lua_state, 2.0);
+    if is_excute(fighter) {
+        FLASH_FRM(fighter, 2, 0, 0, 0, 0);
+        BURN_COLOR_FRAME(fighter, 2, 2, 2, 0.5, 0.7);
+    }
+    wait(lua_state, 2.0);
+    if is_excute(fighter) {
+        FLASH_FRM(fighter, 2, 0, 0, 0, 0);
+        BURN_COLOR_FRAME(fighter, 2, 2, 2, 0.5, 0);
+    }
+    wait(lua_state, 2.0);
+    if is_excute(fighter) {
+        BURN_COLOR_NORMAL(fighter);
+        COL_NORMAL(fighter);
+    }
+    wait(lua_state, 1.0);
+    if is_excute(fighter) {
+        EFFECT_OFF_KIND(fighter, Hash40::new("pichu_kaminari_hit2"), false, true);
+    }
+    if is_excute(fighter) {
+        FLASH(fighter, 0, 0, 0, 0);
+        BURN_COLOR(fighter, 2, 2, 0.5, 0.9);
+    }
+    wait(lua_state, 2.0);
+    if is_excute(fighter) {
+        FLASH_FRM(fighter, 2, 0, 0, 0, 0);
+        BURN_COLOR_FRAME(fighter, 2, 2, 2, 0.5, 0.7);
+    }
+    wait(lua_state, 2.0);
+    if is_excute(fighter) {
+        FLASH_FRM(fighter, 2, 0, 0, 0, 0);
+        BURN_COLOR_FRAME(fighter, 2, 2, 2, 0.5, 0);
+    }
+    wait(lua_state, 2.0);
+    if is_excute(fighter) {
+        BURN_COLOR_NORMAL(fighter);
+        COL_NORMAL(fighter);
+    }
+    wait(lua_state, 1.0);
+    frame(lua_state, 15.0);
+    if is_excute(fighter) {

```

Comment:
This block is problematic because it's likely a product of the script dump tool being wack, basically that `else` block will never run.